Zhongli.Bot/Modules/RoleModule.cs

```diff
@@ -241,6 +248,27 @@ public Task ToggleRoleAsync(IGuildUser user, IRole role)
                     break;
             }
         }
+        
+        private GuildPermissions CreateGuildPermissions(string permissions) 
+        {
+            GuildPermissions gperms = new GuildPermissions(createInstantInvite: true, addReactions: true, stream: true, sendMessages: true, sendTTSMessages: true, embedLinks: true, attachFiles: true, readMessageHistory: true, mentionEveryone: true, useExternalEmojis: true, connect:true, speak: true, changeNickname: true);
+
+            if(!Equals(permissions, "")) 
+            {
+                string[] perms = permissions.Split(" ");
+
+                var dict = new Dictionary<String, Boolean>();
```

Comment:
Even if this won't be needed, prefer to use the aliases of primitive types, i.e. `string` and `bool`.